ee/ucp/admin/install/cloudproviders/install-on-azure.md

```diff
@@ -56,6 +56,19 @@ You must meet the following infrastructure prerequisites to successfully deploy
   needed as part of the UCP prerequisites. If you are using a separate Resource
   Group for the networking components, the same Service Principal will need
   `Network Contributor` access to this Resource Group.
+- Kubernetes pods integrate into the underlying Azure networking stack, from
+  an IPAM and routing perspective with the Azure CNI IPAM module. Therefore
+  Azure Network Security Groups (NSG) impact pod to pod communication. End users
+  may expose containerized services on a range of underlying ports, resulting in
+  a manual process to open an NSG port every time a new containerized service is
+  deployed on to the platform. This would only affect workloads deployed on to
+  the Kubernetes orchestrator. It is advisable to have an "open" NSG between
+  all IPs on the Azure Subnet passed into UCP at [install time](#install-ucp).
+  To limit exposure, this Azure subnet should be locked down to only be used
+  for Container Host VMs and Kubernetes Pods.  Additionally, end users can
+  leverage [Kubernetes Network
+  Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+  to provide micro segmentation for containerized applications and services.
 
```

Comment:
Suggest changing to: "This Azure subnet should be locked down to only be used for Container Host VMs and Kubernetes Pods to limit exposure. Additionally, users can take advantage of..."